src/Workspaces/Remote/Core/Diagnostics/DiagnosticComputer.cs

```diff
@@ -124,28 +165,109 @@ private static ImmutableArray<DiagnosticAnalyzer> GetAnalyzers(BidirectionalMap<
             return builder.ToImmutable();
         }
 
-        private static BidirectionalMap<string, DiagnosticAnalyzer> CreateAnalyzerMap(Project project)
+        private sealed class CompilationWithAnalyzersData
```

Comment:
>class [](start = 23, length = 5)

`readonly struct`?

Reply:
Needs to be a reference type to be a value for CWT.

Reply:
True, missed that.